core/includes/theme.inc

```diff
@@ -1515,7 +1515,7 @@ function theme_has_settings($theme) {
 function theme_has_color_support($theme) {
   $themes = list_themes();
   $theme_info = $themes[$theme];
-  if ($theme_info->info['settings']['color'] && file_exists(backdrop_get_path('theme', $theme_info->name) . '/color/color.inc')) {
+  if (!empty($theme_info->info['settings']['color']) && file_exists(backdrop_get_path('theme', $theme_info->name) . '/color/color.inc')) {
```

Comment:
Shouldn't we also be checking whether color setting is set to `TRUE`? ...something like this perhaps:

```suggestion
  $theme_has_color_settings = isset($theme_info->info['settings']['color']) ? $theme_info->info['settings']['color'] : FALSE;
  $theme_has_color_inc = file_exists(backdrop_get_path('theme', $theme_info->name) . '/color/color.inc');
  if ($theme_has_color_settings && $theme_has_color_inc) {
```

Reply:
> Shouldn't we also be checking whether color setting is set to TRUE?

That's what `!empty($theme_info->info['settings']['color'])` does. If it's set to TRUE/1, `!empty()` returns TRUE. If it's set to FALSE/0/not set at all, `!empty()` returns FALSE.

Reply:
I agree with @BWPanda that `!empty()` does the right check here in function theme_has_color_support().

Reply:
👍